edx_solutions_api_integration/users/serializers.py

```diff
@@ -178,6 +178,15 @@ class CourseProgressSerializer(serializers.Serializer):
     def get_progress(self, enrollment):
         completion_percentage = 0
 
+        # If this enrollment has a value from the new API, use that instead
+        # of calculating one. Note: Must multiply by 100 because its a ratio.
+        if enrollment['course_id'] in self.context['new_api_data']:
+            data_dict = self.context['new_api_data'][enrollment['course_id']]['completion']
+            earned = data_dict['earned']
+            possible = data_dict['possible']
+            completion_percentage = earned * 100.0 / possible
```

Comment:
Is there any benefit of calculating the percentage like this over just multiplying the `ratio` value returned by the new api with 100? It doesn't really matter much, I'm just curious.

Reply:
It's not a ratio that is based on 100. It is ratio between the number of total possible and how many were earned. This needs to be changed with a linear proportion to 100.